apps/web/pages/video/[uid].tsx

```diff
@@ -280,7 +313,10 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       ...(typeof bookingObj.references[0].meetingPassword === "string" && {
         meetingPassword: bookingObj.references[0].meetingPassword,
       }),
-      booking: bookingObj,
+      booking: {
+        ...bookingObj,
+        ...(bookingObj.description && { description: md.render(bookingObj.description) }),
```

Comment:
render markdown in serverside rather than the client side

Reply:
@PeerRich A booking description is not a smart text editor, why are we rendering it in markdown here? 🤔

Reply:
no clue

Reply:
it is markdown